src/rustdoc_schema.graphql

```diff
@@ -483,6 +499,7 @@ type PlainVariant implements Item & Variant {
 
   # edges from Variant
   field: [StructField!]
+  discriminant: Discriminant
```

Comment:
Let's make sure to mirror the doc comment for all these edges.

Unfortunately, GraphQL LSP isn't smart enough to fall back to the supertype's definition of an edge if the concrete type's definition doesn't offer a doc comment. Duplicating is annoying, but it's less annoying than trying to use something that doesn't have docs.